libc/hdr/types/sighandler_t.h

```diff
@@ -0,0 +1,22 @@
+//===-- Definition of macros from __sighandler_t.h ------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_LIBC_HDR_SIGHANDLER_T_H
+#define LLVM_LIBC_HDR_SIGHANDLER_T_H
+
+#ifdef LIBC_FULL_BUILD
+
+#include "include/llvm-libc-types/__sighandler_t.h"
+
```

Comment:
can you add `using sighandler_t = __sighandler_t;` here, and change all the usages to use `sighandler_t` instead of `__sighandler_t`, since the former is the official name defined in `signal.h`.

Reply:
done, please review